### DISCUSSION TITLE
AuthStore is not saving the auth state in Safari

### DISCUSSION THREAD

**Discussion by RRoussev88** [pocketbase/pocketbase#3478] [Q&A] [answered]
After a successful authentication with `authWithPassword` method I want to get the user and the token using `pbClient.authStore.model` and `pbClient.authStore.token` but this isn't working in Safari. Tried on versions 15.6.1 and 17.0 of Safari with version 0.18.0 of pocketbase SDK. 
On further inspection the Cookie store was empty after authenticating.

This flow works well in Chrome, Firefox and Opera

**Comment by ganigeorgiev** [comment, marked as answer]
The default [JS SDK AuthStore](https://github.com/pocketbase/js-sdk#auth-store) doesn't use cookie but LocalStorage.

If you have somewhere `exportToCookie()` and it is not returned in the response, then you can try to export it with `{secure: false}` option since several users reported in the past that Safari doesn't have exception for localhost like the other browsers and require https if `secure` parameter is set.

If that's not the case, please elaborate by providing a code sample and/or reproducible steps.

**Reply by RRoussev88** [reply]
Yes, I was expecting `exportToCookie()` to do the job but it was failing until I added `secure: false` option.
Now all the options I use on export are: `{ httpOnly: false, secure: false }`

Thank you for your timely help!

**Reply by ganigeorgiev** [reply]
_Note that it is recommended to set `secure: false` only during development (you can use env variable to make it dynamic)._
